packages/forms-web-app/src/dynamic-forms/dynamic-components/add-more/index.njk

```diff
@@ -0,0 +1,61 @@
+{% extends layoutTemplate %}
+
+{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
+{% from "govuk/components/radios/macro.njk" import govukRadios %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+
+{% set title= question.pageTitle + " - Appeal a planning decision - GOV.UK" %}
+{% block pageTitle %}{{ "Error: " + title if errors else title }}{% endblock %}
+
+{% block content %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <h1>Hello</h1>
+
+      <ul>
+        {% for answer in addMoreAnswers %}
+            <li>{{answer.label}} :: {{answer.answer}} :: {{answer.removeLink}}</li>
+        {% endfor %}
+      </ul>
+
+      <form method="POST" novalidate>
+        <input type="hidden" name="add-more-question" value="yes"/>
```

Comment:
using a hidden input here to know if we are working on an add more question

Reply:
As an alternative, you could give your `govukButton` below a name i.e.
```
{{ govukButton({
  name: "add-more-question",
  text: "Continue",
  type: "submit",
  attributes: { "data-cy":"button-save-and-continue"}
}) }}
```